src/it/scala/temple/generate/docker/DockerGeneratorIntegrationTest.scala

```diff
@@ -0,0 +1,16 @@
+package temple.generate.docker
+
+import org.scalatest.{BeforeAndAfter, Matchers}
+import temple.containers.HadolintSpec
+
+class DockerGeneratorIntegrationTest extends HadolintSpec with Matchers with BeforeAndAfter {
+
+  behavior of "DockerValidator"
+  it should "fail when a latest tag is used" in {
+    validate("FROM golang:latest\nEXPOSE 80") should include(Rules.useLatest)
+  }
+
+  it should "succeed when a versioned tag is used" in {
+    validate("FROM golang:123") shouldBe empty
```

Comment:
Can this expose 80 for consistency (to avoid adding another source of bugs? It also feels like this should be before the other one